# Detect current vehicle medium.
# Header
imc_ros_interface/Header header
# enumerated list, 0=ground,1=air,2=water,3=underwater,4=unknown
std_msgs/UInt8 medium
